src/vellum/workflows/nodes/displayable/bases/inline_prompt_node/node.py

```diff
@@ -197,6 +198,18 @@ def _process_prompt_event_stream(self) -> Generator[BaseOutput, None, Optional[L
             elif event.state == "STREAMING":
                 yield BaseOutput(name="results", delta=event.output.value)
             elif event.state == "FULFILLED":
+                if (
+                    event.meta
+                    and event.meta.finish_reason == "LENGTH"
+                    and len(event.outputs) == 1
+                    and isinstance(event.outputs[0], StringVellumValue)
```

Comment:
we want to use the logic that prompt nodes use to produce the `text` output instead of these two checks